app/api/login/route.ts

```diff
@@ -10,6 +11,21 @@ export async function POST(request: NextRequest) {
         audience: clientId
     });
     const payload = authResponse.getPayload();
+    const googleUserId = payload["sub"];
+
+    // Setup Redis
+    const redisOptions = {
+        url: process.env.KV_REST_API_URL,
+        token: process.env.KV_REST_API_TOKEN
+    };
+    const redis = new Redis(redisOptions);
+
+    // Post to DB
+    const userDbObj = { googleUserId }
+    const leagueConfigString = JSON.stringify(userDbObj)
```

Comment:
I would say we should put all the db specific interaction in the `dbFetch` module. WDYT?

Reply:
This would probably make the test a bit easier 😅  😬

Reply:
Sure works for me. I'll do that now